recipes/client.rb

```diff
@@ -5,7 +5,7 @@
 
 # Disable Upstart consul service
 service 'consul' do
-  provider Chef::Provider::Service::Upstart
+  provider Chef::Provider::Service::Upstart unless node['platform_version'] == '16.04'
```

Comment:
I think we had a problem with it using init.d scripts by default that was causing some problems, so we hard coded to upstart in 14.04.